packages/ui-components/src/components/DataLibrary/options.js

```diff
@@ -139,18 +167,27 @@ export const SelectableOption = ({ option, isSelected, onSelect, ...restProps })
 
 export const SelectableMultipleTimesOption = ({ option, onSelect }) => (
   <StyledSelectableMultipleTimesOption onClick={onSelect}>
-    <OptionTextWithTooltips option={option} />
+    <Tooltip option={option}>
+      <FlexSpaceBetween>
+        <Option option={option} />
+      </FlexSpaceBetween>
+    </Tooltip>
     <IconWrapper>
       <ChevronRight />
     </IconWrapper>
   </StyledSelectableMultipleTimesOption>
 );
 
-export const SelectedDataCard = ({ option, onRemove, index }) => {
+export const SelectedDataCard = ({ option, index, optionComponent }) => {
   const [isDragging, setIsDragging] = React.useState(false);
+  const [edittingOption, setEdittingOption] = React.useState(null);
 
   return (
-    <Draggable draggableId={option.code} index={index}>
+    <Draggable
+      draggableId={option.code}
+      index={index}
+      isDragDisabled={edittingOption === option.code}
```

Comment:
`edittingOption` to disable dragging ability so that we can drag and select text in editor.